races/android.md

```diff
@@ -1,3 +1,87 @@
 ---
 wrapper: "../../../js/components/wiki/StubArticle"
 ---
+
+
+## Android
+<br><br>
+#### Description<br>
+The androids are robotic and cybernetic units designed for a variety of military and hazardous civilian tasks.  They have limited intelligence but are capable of immense physical strength.  Unlike organic races, androids have no mana.  They are not recommended for inexperienced players.<br><br>
+
+|Starting Stats| |
+|:---|:---|
+|Str | 20 |
```

Comment:
I would suggest using the whole stat name instead of the abbriation, and you can also add an [[tag]] around it to make it automatically link to the wiki article on that stat.